core/src/com/unciv/logic/map/TileInfo.kt

```diff
@@ -55,6 +55,8 @@ open class TileInfo {
     val longitude: Float
         get() = HexMath.getLongitude(position)
 
+    var influenceScore: Float = 0F
```

Comment:
This should be `@Transient` since there's no point in saving the info for the next civ, since it'll just overwrite it